### PR TITLE
Tabular Doc Update

### PR DESCRIPTION
**Pull request by Innixma**
*Issue #, if available:*

#201 

*Description of changes:*

Clarified documentation and logging on tabular class thresholding.
Disabled enable_fit_continuation.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: autogluon/utils/tabular/data/cleaner.py [hunk 1]
```diff
@@ -61,10 +62,10 @@ def get_valid_classes(X, label, threshold):
         sum_after = sum(class_counts_valid)
         percent = sum_after / sum_prior
         if len(valid_classes) < len(class_counts):
-            logger.log(25, 'Warning: Some classes in data were not valid. AutoGluon will only keep '+str(len(valid_classes))+
-                   ' out of '+str(len(class_counts))+ ' classes')
+            logger.log(25, 'Warning: Some classes in the training set have fewer than %s examples. AutoGluon will only keep %s out of %s classes for training. '
+                           'To keep more classes, increase data size or reduce label_count_threshold.' % (threshold, len(valid_classes), len(class_counts)))
```
Comment (jwmueller) [inline autogluon/utils/tabular/data/cleaner.py]: "classes for training"
-> 
"classes for training and will not try to predict the rare classes"

"increase data size" 
-> 
"increase the number of datapoints from these rare classes in the training data"

File: autogluon/utils/tabular/data/cleaner.py [hunk 2]
```diff
@@ -61,10 +62,10 @@ def get_valid_classes(X, label, threshold):
...
         if percent < 1.0:
-            logger.log(25, 'Fraction of data that will be kept for training models: '+str(percent))
+            logger.log(25, 'Fraction of data that contain classes with at least %s examples and will be kept for training models: %s' % (threshold, percent))
```
Comment (jwmueller) [inline autogluon/utils/tabular/data/cleaner.py]: "that contain classes"
-> "from classes"

File: autogluon/utils/tabular/data/cleaner.py (hunk 1 above)
Comment (Innixma) [reply autogluon/utils/tabular/data/cleaner.py]: Updated

File: autogluon/utils/tabular/data/cleaner.py (hunk 2 above)
Comment (Innixma) [reply autogluon/utils/tabular/data/cleaner.py]: Updated

**Comment by szha** [comment]
Job PR-203-3 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-203/3/index.html

**Pull request merged**